src/lib/y2network/widgets/ifplugd_priority.rb

```diff
@@ -0,0 +1,46 @@
+require "cwm/common_widgets"
+
+module Y2Network
+  module Widgets
+    class IfplugdPriority < CWM::IntField
+      def initialize(config)
+        textdomain "network"
+
+        @config = config
+      end
+
+      def label
+        _(
+          "Ifplugd Priority"
```

Comment:
Btw I hate this label. We should not disclosure some ifplugd where noone knows what it is. It is something like "Apply Priority" or stuff like this. Basically if more on cable connection is  there, then this number decide which to apply ( the one which is available and has the lowest number ).